### PR TITLE
Always initalize DateTime in UTC. Fixes #52.

### LINKED ISSUE #52: Make tests independent of the system local
There are many places where `new DateTime(1970, 1, 1, 0, 0, 0)` is used. Some calculation is done with this epoch and then stored in DateTimeOffset. The resulting DateTimeoffset.Offset will be dependent on the timezone settings of the computer.

As a result, the test `SetAttributionInfo_WhenReferrerTimestampSpecified_IsAddedToRequest` fails on my computer.

Any reason why `new DateTime(1970, 1, 1, 0, 0, 0, DateTimeKind.Utc)` isn't used?

### PR DESCRIPTION
**Pull request by huan086**

### CODE REVIEW

**Comment by huan086** [comment]
Sorry for messing up the pull request. Didn't mean to have the async stuff. Fixing it now

File: Piwik.Tracker/PiwikTracker.cs [hunk 1]
```diff
@@ -182,7 +182,7 @@ public class PiwikTracker
         private bool _configCookiesDisabled;
         private string _configCookiePath = DefaultCookiePath;
         private string _configCookieDomain = "";
-        private readonly long _currentTs = (long)(DateTime.Now - new DateTime(1970, 1, 1)).TotalSeconds;
+        private readonly long _currentTs = (long)(DateTime.UtcNow - new DateTime(1970, 1, 1, 0, 0, 0, DateTimeKind.Utc)).TotalSeconds;
```
Comment (huan086) [inline Piwik.Tracker/PiwikTracker.cs]: For consistency, I would initialize Unix epoch as `new DateTime(1970, 1, 1, 0, 0, 0, DateTimeKind.Utc)` always, though it doesn't make any difference in this instance.

Comment (julienmoumne) [inline Piwik.Tracker/PiwikTracker.cs]: I agree with your point. In that case, could we maybe factor out the construction of such a date in a common function to:
- make the code more readable by attributing this date value a name
- promote the usage of UTC

**Comment by julienmoumne** [comment]
This looks brilliant, thanks.

Maybe next time avoid rebasing so that we don't loose the reviews.

Cheers

**Comment by huan086** [comment]
Ok. Interesting, normally at other repo my pull request goes "there's a merge conflict, please rebase".

The code review is still there. Click **show outdated**. Is there any other places you are expecting to see the review?

**Comment by julienmoumne** [comment]
Thanks for taking the time to explain it to me, I did not see the outdated link. Everything's ok. Cheers

**Pull request merged**